### ISSUE TITLE
[Suggestion] Fabric 1.18?

### ISSUE THREAD

**Issue by PoneyClairDeLune** [skyjay1/Nomadic-Tents#63]
Hello, may I ask whether there will be a Fabric 1.18 port of Nomadic Tents?

**Comment by skyjay1** [comment]
No, I do not plan to re-write the mod for Fabric. I am very close to releasing a 1.16.5 Forge version, though
